server/storage/players.lua

```diff
@@ -317,6 +317,7 @@ end
 ---@param citizenid string
 ---@return table<string, integer> jobs
 ---@return table<string, integer> gangs
+---@return table<string, integer> playerGroups
```

Comment:
There is no matching return for this annotation.

Reply:
```suggestion
```